src/range-slider/index.tsx

```diff
@@ -155,9 +155,9 @@ export const RangeSlider = factory(function RangeSlider({
 	const getInputProperties = (isSlider1: boolean) => ({
 		...formatAriaProperties(aria),
 		'aria-describedby': isSlider1 ? minLabelId : maxLabelId,
-		'aria-invalid': valid === false ? 'true' : null,
+		'aria-invalid': (valid === false ? 'true' : undefined) as 'true' | undefined,
```

Comment:
is the `as` needed here?

Reply:
Unfortunately, since TS doesn't infer the correct type here. I've changed it to use the exported `AriaAttributes` interface explicitly though.